### PR TITLE
Logits selection method fixed

### PR DESCRIPTION
**Pull request by yashenkoxciv**

### CODE REVIEW

File: ml/curatedBreastData-InfoGAN/xstudy_test.py [hunk 1]
```diff
@@ -60,8 +60,10 @@ def apply_logistic_regression(train_x, train_y, test_x, test_y):
     test_predictions = logreg.predict(test_x)
     all_train_logits = logreg.predict_proba(train_x)
     all_test_logits = logreg.predict_proba(test_x)
-    train_logits = all_train_logits[range(train_y.size), train_y.astype(np.int)]
-    test_logits = all_test_logits[range(test_y.size), test_y.astype(np.int)]
+    #train_logits = all_train_logits[range(train_y.size), train_y.astype(np.int)]
+    #test_logits = all_test_logits[range(test_y.size), test_y.astype(np.int)]
+    train_logits = all_train_logits[:, 1]
```
Comment (noskill) [inline ml/curatedBreastData-InfoGAN/xstudy_test.py:65]: Good, but what about xgboost evalation?

Comment (noskill) [reply ml/curatedBreastData-InfoGAN/xstudy_test.py:65]: there is bug in the code below:
test_auc = roc_auc_score(test_predictions, test_logits),

should be

test_auc = roc_auc_score(test_y, test_logits)

**Review by noskill — CHANGES_REQUESTED** [review]

**Comment by mjsduncan** [comment]
thanks, @yashenkoxciv !
